rocks/Factory.java

```diff
@@ -48,6 +48,8 @@
 
     interface TransactionSchema {
 
+        RocksTransaction.Schema transactionSchemaInit(RocksSession.Schema session);
```

Comment:
We need transactionSchemaInit in order to provide the hook for cluster to supply an additional logic with regards to schema initialisation.